edualgo/graph_algorithms/Dijkstra_algorithms.py

```diff
@@ -0,0 +1,117 @@
+from .__init__ import print_msg_box
+
+import sys
+
+# Providing the graph
+vertices = [[0, 0, 1, 1, 0, 0, 0],
```

Comment:
We don't need hard coded input for our package, just pass it as a parameter to the function,

Reply:
Done